### ISSUE TITLE
[DataCap Application] NLM Public dataset

### ISSUE THREAD

**Issue by DarkForest3** [filplus-bookkeeping/IPFSCN#35]
### Version

1

### DataCap Applicant

NLM Public dataset

### Project ID

1

### Data Owner Name

National Library of Medicine

### Data Owner Country/Region

United States

### Data Owner Industry

Life Science / Healthcare

### Website

https://www.nlm.nih.gov/portals/healthcare.html

### Social Media Handle

https://www.facebook.com/nationallibraryofmedicine

### Social Media Type

Facebook

### What is your role related to the dataset

Data Preparer

### Total amount of DataCap being requested

4PiB

### Expected size of single dataset (one copy)

1PiB

### Number of replicas to store

4

### Weekly allocation of DataCap requested

512TiB

### On-chain address for first allocation

f14byzc3pazlqyf4qivfwxie6rwjlnm3b6iu2mppy

### Data Type of Application

Public, Open Dataset (Research/Non-Profit)

### Custom multisig

- [ ] Use Custom Multisig

### Share a brief history of your project and organization

```text
The National Library of Medicine (NLM) is a part of the National Institutes of Health in the United States. It is the world's largest biomedical library and provides access to a wide range of medical resources and information. The NLM's website, https://www.nlm.nih.gov/, offers a variety of online resources including medical literature, databases, and research tools for healthcare professionals, researchers, and the general public.
```

### Is this project associated with other projects/ecosystem stakeholders?

No

### Describe the data being stored onto Filecoin

```text
The data available on NLM mainly includes medical literature, biomedical databases, genomic data, medical imaging, and other information. We hope that the data stored on Filecoin includes medical literature and research materials, biomedical databases, and genomic data. These data are of great significance to medical research and practice, and by storing them on Filecoin, more secure, reliable, and decentralized data storage and sharing can be achieved.
```

### Where was the data currently stored in this dataset sourced from

AWS Cloud

### If you are a data preparer. What is your location (Country/Region)

China

### If you are a data preparer, how will the data be prepared? Please include tooling used and technical details?

```text
We have developed our own website data download tool and collected it. And send data to SP according to their needs, including sending hard drives, network transmission.
```

### Please share a sample of the data

```text
aws s3 ls --no-sign-request s3://sra-pub-sars-cov2/
aws s3 ls --no-sign-request s3://sra-pub-sars-cov2-metadata-us-east-1/
```

### Confirm that this is a public dataset that can be retrieved by anyone on the Network

- [X] I confirm

### What is the expected retrieval frequency for this data

Monthly

### For how long do you plan to keep this dataset stored on Filecoin

2 to 3 years

### In which geographies do you plan on making storage deals

Greater China, Asia other than Greater China

### How will you be distributing your data to storage providers

Cloud storage (i.e. S3), HTTP or FTP server, IPFS, Shipping hard drives, Lotus built-in data transfer

### How did you find your storage providers

Slack, Partners

### Please list the provider IDs and location of the storage providers you will be working with.

```text
f01084941 HK
f01084413 Singapore
f03228820  KR
f01082888  HK
f03081958  China
f03253401  China
f03238633  Singapore
f01531188  HK
f03241759  China
f03241413  China
```

### How do you plan to make deals to your storage providers

Lotus client

### Can you confirm that you will follow the Fil+ guideline

Yes

**Comment by DarkForest3** [comment]
As the applicant, I will ensure that all necessary security measures are taken during the data migration process. For hard drives sent by mail, we will use encrypted hard drives and ensure physical security during transportation. For network transmission, we will use SSL/TLS encryption to protect the security of data transmission. We have already negotiated specific storage and transmission protocols with the storage providers to ensure they understand our data security requirements and use the Lotus client to manage the storage and retrieval process of the data. Furthermore, we will regularly conduct data integrity checks to ensure the accuracy and reliability of the data stored on the Filecoin network.

**Comment by ipfscn** [comment]
Is the data ready now? Can we start sealing?

**Comment by DarkForest3** [comment]
Yes, most of the data has been prepared. We also sent the hard drive to SP in advance. Now we can start sealing. The remaining data will be provided to SP gradually.

**Comment by lijo76618** [comment]
checker:manualTrigger

**Comment by DarkForest3** [comment]
checker:manualTrigger

**Comment by DarkForest3** [comment]
I hope you can provide support again. Thank you

**Comment by filecoin-watchdog** [comment]
checker:manualTrigger

**Comment by DarkForest3** [comment]
Hello, respected governance team.
After customers apply to us, we will ask them on GitHub:
1. Technical solutions and details.
2. Does it support Spark retrieval.
We have a total of three clients in this round, and before assigning them, we asked them all.
Taking the first customer as an example:
https://github.com/filplus-bookkeeping/IPFSCN/issues/31
We asked them four questions in total before signing for them twice.
Two technical issues, one is the Spark issue, and the other is whether it can start immediately (time).
![image](https://github.com/user-attachments/assets/ddea4b6b-f996-4067-a039-e4ca645425f5)

We only signed after they all replied.
First signature 256T
Second signature 512T
Third signature 1PiB
I checked the bot data before the second and third signatures:
The disclosed SP and the actual cooperating SP are the same.
Most SPs support Spark.
There were no issues such as CID or duplicate data.
After the third signature, we found that the client had added a new SP, but they did not disclose it in advance.
And we found that these SPs may have used VPN.
So, we asked again. Although we have seen their answers, we may no longer continue to support them.
![image](https://github.com/user-attachments/assets/cbd566ad-cbe8-4bbb-961f-d9e3af535565)
Next, we will scrutinize our clients more rigorously. We will also work harder to carry out our work. Thank you

**Comment by DarkForest3** [comment]
This is my problem, I didn't pay attention to and report the addition of the new SP in a timely manner. I will pay attention to this issue in the future.

**Comment by DarkForest3** [comment]
Dear allocator, we have updated the SP for our collaboration and request your support.
We hope to complete the sealing work before New Year's Day, as the daily rental fee for the machines we rent is expensive.
f01084941 HK
f01084413 Singapore
f03228820  KR
f01082888  HK
f03081958  China
f03253401  China
f03238633  Singapore
f01531188  HK
f03241759  China
f03241413  China

**Comment by ipfscn** [comment]
Why does the SP list keep changing?

**Comment by DarkForest3** [comment]
At present, our project lasts for a long time, and some SPs have changed their sealing plans. We have to find another SP to cooperate with. It is not easy to find SPs now. Most SPs are relatively small, and few can provide long-term stable support.

**Comment by ipfscn** [comment]
Could you please introduce yourself and elaborate on how you prepared the data?

**Comment by DarkForest3** [comment]
Of course. I am an independent data preparer from China. You can call me sam. I think fil's solution is a very good backup method for the security of public data. So, currently, I am committed to collecting valuable public data and storing it in the fil network.
The process of data preparation is roughly like this. I first determine the source of data to be stored. For NLM, I will first develop a crawler program to traverse the website and find valuable data links. For example, the following https://ftp.ncbi.nlm.nih.gov/100GB. Prepare local data storage and download the data, and then choose to package or split it according to the size of the data. Most of the data will be copied to the hard disk and sent to the SP. The packaging work is completed by the SP.

**Comment by ipfscn** [comment]
Can your newly added SPs support Spark normally? Are you ready to start sealing?

**Comment by DarkForest3** [comment]
Yes, all SPs support retrieval normally. And now it is also ready for sealing. Thank you for your support

**Comment by ipfscn** [comment]
checker:manualTrigger

**Comment by ipfscn** [comment]
Several SPs have a retrieval rate of zero. And I found undisclosed SPs. For example, f02088691. Please provide an explanation for this.
